cmake/kokkos_tribits.cmake

```diff
@@ -116,6 +116,13 @@ function(KOKKOS_ADD_EXECUTABLE_AND_TEST ROOT_NAME)
       target_compile_options(${PACKAGE_NAME}_${ROOT_NAME} PRIVATE "-fno-rtti")
     endif()
   endif()
+  if(KOKKOS_ENABLE_NO_EXCEPTIONS AND NOT KOKKOS_CXX_COMPILER_ID STREQUAL "NVIDIA")
```

Comment:
Please add a comment that it does not work with CUDA on Microsoft Windows and that the Thrust library that we use in particular for sorting algorithms on the device with CUDA does not support building without exceptions as of today